modules/extensions-spring-support/src/test/resources/models/heisenberg.json

```diff
@@ -33351,7 +33351,7 @@
           "namespace": "MULE"
         }
       },
-      "minMuleVersion": "4.1.1",
+      "minMuleVersion": "4.5.0",
```

Comment:
Method approve has min mule version 4.5.0 because of its parameter recursivePojo. Parameter recursivePojo has min mule version 4.5.0 because it is of type RecursivePojo. Type RecursivePojo has min mule version 4.5.0 because of its field next. Field next has min mule version 4.5.0 because it is a parameter of type RecursivePojo. Type RecursivePojo has min mule version 4.5.0 because it is annotated with TypeDsl. TypeDsl was introduced in Mule 4.5.0.